docs/quick-start-guide.md

```diff
@@ -50,6 +50,43 @@ $ kubectl describe deployment sparkoperator -n sparkoperator
 
 ```
 
+### Metrics
+
+The operator exposes a set of metrics via the metric endpoint to be scraped by `Prometheus`.  If you would like to expose metrics, please install the operator via `kubectl apply -f manifest/spark-operator-with-metrics.yaml`.
+This installs the operator with additional flag to enable metrics (`-enable-metrics=true`) as well as other annotations used by Prometheus to scrape the metric endpoint.
+  
+If enabled, the operator generates the following metrics:
+
+| Metric | Description |
+| ------------- | ------------- |
+| `spark_app_submit_count`  | Total number of SparkApplication submitted by the Operator.|
+| `spark_app_success_count` | Total number of SparkApplication which completed successfully.|
+| `spark_app_failure_count` | Total number of SparkApplication which failed to complete. |
+| `spark_app_running_count` | Total number of SparkApplication which are currently running.|
+| `spark_app_success_execution_time_microseconds` | Execution time for applications which succeeded.|
+| `spark_app_failure_execution_time_microseconds` |Execution time for applications which failed. |
+| `spark_app_executor_success_count` | Total number of Spark Executors which completed successfully. |
+| `spark_app_executor_failure_count` | Total number of Spark Executors which failed. |
+| `spark_app_executor_running_count` | Total number of Spark Executors which are currently running. |
+
+The following is a list of all the configurations the operators supports for metrics: 
+
+```bash
+-enable-metrics=true
+-metrics-port=10254
```

Comment:
It's worth mentioning that if users use a different port, they also need to update the annotation `prometheus.io/port` and `containerPort` in the yaml.

Reply:
sure, done